### PR TITLE
support for empty datasets

### PR DESCRIPTION
**Pull request by gowerc**
Address the most basic part of #44

### CODE REVIEW

File: R/diffdf.R [hunk 1]
```diff
@@ -80,6 +80,7 @@ diffdf <- function (
     tolerance = sqrt(.Machine$double.eps),
     scale = NULL
 ){
+    #browser()
```
Comment (kieranjmartin) [inline R/diffdf.R]: this should be deleted

Comment (gowerc) [reply R/diffdf.R]: Fixed

**Comment by gowerc** [comment]
Hey @kieranjmartin ,

Addressed your comment and added a few more tests. Could you review and merge when you get a chance please !

**Comment by gowerc** [comment]
Ported main code changes + tests into #67

**Pull request closed without merging**